<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgOpenGlSettings</class>
 <widget class="QDialog" name="CQDlgOpenGlSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenGL settings</string>
  </property>
  <widget class="QLabel" name="qqText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>If you are experiencing rendering problems with vision sensors or the regular visualization (e.g. empty frames, low framerates or crashes), then adjusting settings below might help.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="qqOk">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>350</y>
     <width>351</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>161</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Offscreen context type</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QRadioButton" name="qqContextDefault">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Default (recommended)</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="qqContextOffscreen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Offscreen surface</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="qqContextVisibleWindow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Visible window</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="qqContextInvisibleWindow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Invisible window</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>181</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>FBO type</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QRadioButton" name="qqFboDefault">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Default (recommended)</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="qqFboNonQt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Non-Qt</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="qqFboQt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>171</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Qt</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Various settings</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Idle fps</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqIdleFps">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="qqText_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>If your mouse doesn't operate as expected during object selection
operations, make sure to load 'mouseTestScene' in your 'scenes'-folder.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>181</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>VBO operation</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QRadioButton" name="qqVboDefault">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Default (recommended)</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="qqVboEnabled">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Enabled (when available)</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="qqVboDisabled">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>171</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Disabled</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqContextDefault</tabstop>
  <tabstop>qqContextOffscreen</tabstop>
  <tabstop>qqContextVisibleWindow</tabstop>
  <tabstop>qqContextInvisibleWindow</tabstop>
  <tabstop>qqFboDefault</tabstop>
  <tabstop>qqFboQt</tabstop>
  <tabstop>qqFboNonQt</tabstop>
  <tabstop>qqVboDefault</tabstop>
  <tabstop>qqVboDisabled</tabstop>
  <tabstop>qqVboEnabled</tabstop>
  <tabstop>qqIdleFps</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
